cmd/tigerblood/main.go

```diff
@@ -4,11 +4,23 @@ import (
 	"fmt"
 	"github.com/DataDog/datadog-go/statsd"
 	"go.mozilla.org/tigerblood"
+	"gopkg.in/yaml.v2"
+	"io/ioutil"
 	"log"
 	"net/http"
 	"os"
 )
 
+func loadCredentials() (map[string]string, error) {
+	creds := make(map[string]string)
+	bytes, err := ioutil.ReadFile("credentials.yml")
```

Comment:
add a `-f` flag to read the path to the configuration file on the cmdline, and use a default if not provided